internal/services/redis/redis_cache_resource_test.go

```diff
@@ -720,10 +733,13 @@ resource "azurerm_redis_cache" "test" {
   enable_non_ssl_port = false
 
   redis_configuration {
-    rdb_backup_enabled = false
+    rdb_backup_enabled              = false
+    aof_backup_enabled              = true
+    aof_storage_connection_string_0 = "DefaultEndpointsProtocol=https;BlobEndpoint=${azurerm_storage_account.test.primary_blob_endpoint};AccountName=${azurerm_storage_account.test.name};AccountKey=${azurerm_storage_account.test.primary_access_key}"
+    aof_storage_connection_string_1 = "DefaultEndpointsProtocol=https;BlobEndpoint=${azurerm_storage_account.test.primary_blob_endpoint};AccountName=${azurerm_storage_account.test.name};AccountKey=${azurerm_storage_account.test.secondary_access_key}"
```

Comment:
there's `primary_` and `secondary_` connection strings available on the storage account resource?

Reply:
This shouldn't be the `primary` and `secondary` connection strings for the same storage account. This should be 2 separate storage accounts. 

Ref: https://learn.microsoft.com/en-us/azure/azure-cache-for-redis/cache-how-to-premium-persistence#when-should-i-use-a-second-storage-account

Reply:
@tombuildsstuff , @mclacore , thanks for your feedback. I have updated the test. Could you take another look?

Reply:
LGTM 👍